README.md

```diff
@@ -458,6 +458,28 @@ Exports a **Daru::DataFrame** into a **.xls** file.
     df.write_excel('path/to/file.xls', header: {color: :red, weight: :bold}, data: {color: :blue }, index: false)
     ```
 
+### Excelx Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+Exports a **Daru::DataFrame** into a **.xlsx** file.
+
+- **Docs**: [rubydoc.info](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/Excelx)
+- **Gem Dependencies**: `rubyXL` gem
+- **Usage**:
+    ```ruby
+    #! Partially require just Excelx Exporter
+    require 'daru/io/exporters/excelx'
+
+    #! Usage from Daru::IO
+    string = Daru::IO::Exporters::Excelx.new(df, index: false).to_s
+    Daru::IO::Exporters::Excelx.new(df, index: false).write('path/to/file.xlsx')
+
+    #! Usage from Daru::DataFrame
+    string = df.to_excelx_string(index: false)
+    df.write_excel('path/to/file.xlsx', index: false)
+    ```
+
```

Comment:
I am starting to think this part of README becames too long and repetitive. Just as an idea for the future:
1. Leave 2-3 most "tasty" examples in REDME + link to...
2. Formats.md with regular structure...
3. Which is built automatically, gathering the comments from top of all exporters/importers files, with similar format.

WDYT?

Reply:
For generating files like `{FORMAT}_Importer.md`, does an ERB template work? Or is there a better option?

Reply:
Let's think. In my head, it (probably) does the following:
* takes all the importers (exporters) code;
* extract, say, top-of-the-file comment with explanations;
* join them together with pretty headers;
* store to `Formats.md` (one, not document-per-formatter, it is tiresome to browse).
* + probably generates TOC on the top of the file;
* + probably adds some "preface" at the top of the file (taken from some `_preface.md`).

Can be implemented with just Ruby code, or small ERB template, which is the simplest.

WDYT?

Reply:
@zverok - Acknowledged. Let us take this discussion to #71 please? 😄